tests/testthat/_snaps/install.md

```diff
@@ -3,15 +3,14 @@
     Code
       install("bread@0.1.0")
     Output
-      Downloading 1 package(s) and their dependencies
-      Done
+      # Downloading package(s) =============
```

Comment:
Let's change the format of `header()` here, the `#` and `=` feel too big and noisy. (use `--`?)

Reply:
Yeah, I was thinking I'd do that in a follow up PR.